src/System.Net.Mail/tests/Functional/LoggingTest.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Net.Mail.Tests
     public class LoggingTest : RemoteExecutorTestBase
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
```

Comment:
>Nit: This attribute can be applied to the whole class rather than to each individual method.

DId not know that.  Thx.  I will make a note for next time.